Gadget.js

```diff
@@ -128,6 +128,17 @@
         if (!hasData) {
             return;
         }
+        // sort IPs and UAs
+        $.each(data, function(idx){
+        	ip = data[idx].ip;
+        	ip.sort((a, b) => {
```

Comment:
Arrow functions (and ES6 in general) is not supported in IE11: https://caniuse.com/#feat=arrow-functions
IE11 is a grade A browser that we must support for JS code: https://www.mediawiki.org/wiki/Compatibility#Browsers

Reply:
I convert it into a standalone function.